go.mod

```diff
@@ -11,19 +11,19 @@ require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.9.0
-	github.com/containers/buildah v1.19.7
-	github.com/containers/common v0.35.0
+	github.com/containers/buildah v1.19.2-0.20210305112159-898eea8fbefd
```

Comment:
Shouldn't this be 1.20 something?  I saw the commit in master, maybe  I missed the 1.19 commit?

Reply:
We have not created a 1.20 yet, so I just put in the current commit id.  Since we have not updated the main branch beyond v1.19.2 this is what it show up as.
IE I set the version to
898eea8fbefd73993406610311cad8fcab045bad

And then make vendor-in-container changes it to the above.  BTW I think it might be time to cut a new buildah release.